components/TokensGrid.tsx

```diff
@@ -6,13 +6,16 @@ import useTokens from '../hooks/useTokens'
 import SwapCartModal from 'components/SwapCartModal'
 import TokenCard from './TokenCard'
 import { Token } from 'recoil/cart/atom'
+import { Collection } from 'types/reservoir'
+import { useCollections } from '@reservoir0x/reservoir-kit-ui'
```

Comment:
Is this needed? Doesn't seem like it's being used.

Reply:
👍 Good catch. Just deleted the unused import